cassdegrees/templates/widgets/staff/courselistingwidget.html

```diff
@@ -17,6 +17,11 @@
 
 <h4>Course listing</h4>
 <div id={{ app_name }}>
+    <div style="text-align:right; width:100%; padding:0; padding-bottom: 5px">
```

Comment:
Does this need to be hardcoded here? Would be nice if this could be extracted to CSS